cli/core/pkg/auth/tkg/cluster_pinniped_info.go

```diff
@@ -114,7 +102,7 @@ func GetClusterInfoFromCluster(clusterAPIServerURL, configmapName string) (*clie
 // 'discoveryPort' is used to optionally override the port used for discovery. This may be needed on setups that expose
 // discovery information to unauthenticated users on a different port (for instance, to avoid the need to anonymous auth
 // on the apiserver). By default, the endpoint from the cluster-info is used.
-func GetPinnipedInfoFromCluster(clusterInfo *clientcmdapi.Cluster, discoveryPort *int) (*PinnipedConfigMapInfo, error) {
+func GetPinnipedInfoFromCluster(clusterInfo *clientcmdapi.Cluster, discoveryPort *int) (*pinnipedinfo.PinnipedInfo, error) {
```

Comment:
This function is a candidate for our next step in refactoring as `GetPinnipedInfoFromCluster()` is duplicated.